tests/cloud/deployment/google/test_cloud_run.py

```diff
@@ -78,12 +78,13 @@ def test_with_no_deployment_configuration_file(self):
         with mock.patch.dict(os.environ, {"SERVICE_ID": self.EXAMPLE_SERVICE_ID}):
             with mock.patch("octue.cloud.deployment.google.cloud_run.Runner") as mock_runner:
                 with mock.patch("octue.cloud.pub_sub.service.Topic", new=MockTopic):
-                    cloud_run.answer_question(
-                        project_name="a-project-name",
-                        data={},
-                        question_uuid="8c859f87-b594-4297-883f-cd1c7718ef29",
-                        credentials_environment_variable="GOOGLE_APPLICATION_CREDENTIALS",
-                    )
+                    with mock.patch("octue.cloud.deployment.google.cloud_run.Service"):
```

Comment:
Next time you touch this, suggest creating a reusable context manager that patches everything as necessary.